sdk/vppagent/dataplane/empty_dataplane_server.go

```diff
@@ -0,0 +1,26 @@
+package dataplane
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/crossconnect"
+	"github.com/networkservicemesh/networkservicemesh/dataplane/pkg/apis/dataplane"
+	"github.com/networkservicemesh/networkservicemesh/sdk/vppagent/dataplane/state"
+)
+
+type EmptyChainedDataplaneServer struct {
```

Comment:
Lets not do inheritance here.  A Chain element that implements the interface should work whether or not it derives from any particular ChainedDataplane including the EmptyChainedDataplane.

Reply:
Or to be more clear... this approach should *never* require inheritance to work.  Please see the example in the SDK work for NSEs.

Reply:
thanks, fixed